apps/video/test/marionette/lib/video.js

```diff
@@ -192,6 +197,58 @@ Video.prototype = {
   },
 
   /**
+   * Taps 'pause' button and checks that playback has stopped.
+   * returns false if the icon is not changed.
+   */
+  pausePlayback: function() {
+    this.client.helper.waitForElement(Video.Selector.playpauseButton).click();
+
+    // wait for the icon to change by checking for play icon
+    return this.client.helper.waitForElement(Video.Selector.playIcon)
+    .displayed();
+  },
+
+  /**
+   * Taps 'play' button and checks that the counter is progressing.
```

Comment:
The function is not checking that the elapsed time is incrementing, although I think that is the proper functionality. Probably best to add the 'waitFor' statement similar to how you've done with 'tapForward'.